themes/src/main/resources-community/theme/base/account/theme.properties

```diff
@@ -1 +1 @@
-locales=ar,ca,cs,da,de,el,en,es,fa,fr,fi,hu,it,ja,lt,nl,no,pl,pt,pt-BR,ru,sk,sv,th,tr,uk,zh-CN,zh-TW
+locales=ar,ca,cs,da,de,el,en,es,fa,fr,fi,hu,it,ja,lt,lv,nl,no,pl,pt,pt-BR,ru,sk,sv,th,tr,uk,zh-CN,zh-TW
```

Comment:
Interesting, we were not including Latvian before?

Reply:
It was present in the login locales list, but not on the account locales list. I double checked and the files `messages_lv.properties` contains data, so I added it. And without it, the dropdown list in the locales for the Account UI was only showing the fallback, not the message properties entry.